.drone/drone.yml

```diff
@@ -0,0 +1,61 @@
+---
+kind: pipeline
+type: docker
+name: docker-push
+
+trigger:
```

Comment:
```yaml
trigger:
  branch:
    - grafana
  event:
    exclude:
      - pull_request
      - tag
    include:
      - push
```

^^ the above will limit the workflow only to `push` events on the `grafana` branch. Otherwise it would run on any branch.